src/Microsoft.DotNet.Build.Tasks/VersionTools/BaseDependenciesTask.cs

```diff
@@ -298,6 +319,11 @@ private FileRegexUpdater ConfigureFileRegexUpdater(FileRegexUpdater updater, ITa
                 updater.Regex = new Regex(manualRegex);
                 updater.VersionGroupName = GetRequiredMetadata(step, "VersionGroupName");
             }
+            else if (!string.IsNullOrEmpty(msbuildSdkName))
+            {
+                updater.Regex = CreateMSBuildSdkUpdateRegex(Regex.Escape(msbuildSdkName), nameof(msbuildSdkName));
+                updater.VersionGroupName = nameof(msbuildSdkName);
+            }
             else
             {
                 throw new ArgumentException(
```

Comment:
Could you add `MSBuildSdkName` to the exception message?

Reply:
Fixed.